storage/fio-test-setup.yaml

```diff
@@ -44,6 +44,9 @@
   - name: Create directory for fio resources
     file: path=/tmp/origin state=directory
 
+  - name: Copy clear cache
```

Comment:
Why do we need this task?

Reply:
@hongkailiu when I pushed my changes for some reason the merge removed the --pre-iteration-script option from the fio-pbench job. This is to provide an executable script.
    Will update it and re-submit the PR.